#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_pdftex-EC-encoding by Rafael Laboissiere <rafael@debian.org>
##
## DP: Use ec.enc file instead of EC.enc, which does not exist in Debian

@DPATCH@

--- plplot-5.8.0.orig/doc/docbook/src/pdftex.map
+++ plplot-5.8.0/doc/docbook/src/pdftex.map
@@ -1,31 +1,31 @@
-cmr10 "ECEncoding ReEncodeFont"<EC.enc <cmr10.pfb
+cmr10 "ecEncoding ReEncodeFont"<ec.enc <cmr10.pfb
 cmmi10 <cmmi10.pfb
 cmsy10 <cmsy10.pfb
 cmsy9 <cmsy9.pfb
-ectt0800 "ECEncoding ReEncodeFont"<EC.enc <cmtt8.pfb
-ecit0900 "0.9 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmitt10.pfb
-ecit1000 "ECEncoding ReEncodeFont"<EC.enc <cmitt10.pfb
-tcrm0800 "ECEncoding ReEncodeFont"<EC.enc <cmr8.pfb
-ecrm0600 "ECEncoding ReEncodeFont"<EC.enc <cmr8.pfb
-ectt0900 "ECEncoding ReEncodeFont"<EC.enc <cmtt9.pfb
-ecti1000 "ECEncoding ReEncodeFont"<EC.enc <cmti10.pfb 
-ecbx1000 "ECEncoding ReEncodeFont"<EC.enc <cmbx10.pfb
-ecrb1000 "ECEncoding ReEncodeFont"<EC.enc <cmbx10.pfb
-ecrm0800 "ECEncoding ReEncodeFont"<EC.enc <cmr8.pfb
-ecti0800 "ECEncoding ReEncodeFont"<EC.enc <cmti8.pfb
-tcrm1000 "ECEncoding ReEncodeFont"<EC.enc <cmr10.pfb
-ecrm1000 "ECEncoding ReEncodeFont"<EC.enc <cmr10.pfb
-ecrm0900 "ECEncoding ReEncodeFont"<EC.enc <cmr10.pfb
-ecsx1000 "ECEncoding ReEncodeFont"<EC.enc <cmssbx10.pfb
-ecss1000 "ECEncoding ReEncodeFont"<EC.enc <cmssbx10.pfb
-ecsx1200 "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecss1200 "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecsx1440 "1.2 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecss1440 "1.2 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecsx1728 "1.44 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecss1728 "1.44 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss12.pfb
-ecsx2074 "1.0165 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss17.pfb
-ecss2074 "1.0165 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss17.pfb
-ecsx2488 "1.4635 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss17.pfb
-ecss2488 "1.4635 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmss17.pfb
-ectt1095 "1.095 ExtendFont" "ECEncoding ReEncodeFont"<EC.enc <cmtt10.pfb
+ectt0800 "ecEncoding ReEncodeFont"<ec.enc <cmtt8.pfb
+ecit0900 "0.9 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmitt10.pfb
+ecit1000 "ecEncoding ReEncodeFont"<ec.enc <cmitt10.pfb
+tcrm0800 "ecEncoding ReEncodeFont"<ec.enc <cmr8.pfb
+ecrm0600 "ecEncoding ReEncodeFont"<ec.enc <cmr8.pfb
+ectt0900 "ecEncoding ReEncodeFont"<ec.enc <cmtt9.pfb
+ecti1000 "ecEncoding ReEncodeFont"<ec.enc <cmti10.pfb 
+ecbx1000 "ecEncoding ReEncodeFont"<ec.enc <cmbx10.pfb
+ecrb1000 "ecEncoding ReEncodeFont"<ec.enc <cmbx10.pfb
+ecrm0800 "ecEncoding ReEncodeFont"<ec.enc <cmr8.pfb
+ecti0800 "ecEncoding ReEncodeFont"<ec.enc <cmti8.pfb
+tcrm1000 "ecEncoding ReEncodeFont"<ec.enc <cmr10.pfb
+ecrm1000 "ecEncoding ReEncodeFont"<ec.enc <cmr10.pfb
+ecrm0900 "ecEncoding ReEncodeFont"<ec.enc <cmr10.pfb
+ecsx1000 "ecEncoding ReEncodeFont"<ec.enc <cmssbx10.pfb
+ecss1000 "ecEncoding ReEncodeFont"<ec.enc <cmssbx10.pfb
+ecsx1200 "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecss1200 "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecsx1440 "1.2 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecss1440 "1.2 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecsx1728 "1.44 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecss1728 "1.44 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss12.pfb
+ecsx2074 "1.0165 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss17.pfb
+ecss2074 "1.0165 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss17.pfb
+ecsx2488 "1.4635 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss17.pfb
+ecss2488 "1.4635 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmss17.pfb
+ectt1095 "1.095 ExtendFont" "ecEncoding ReEncodeFont"<ec.enc <cmtt10.pfb
